plugins/modules/aci_rest.py

```diff
@@ -303,6 +309,28 @@ def update_qsl(url, params):
         return url + "?" + "&".join(["%s=%s" % (k, v) for k, v in params.items()])
 
 
+def add_annotation(annotation, payload):
```

Comment:
Are we setting annotation in child objects in the payload?

Reply:
I thought about it but didn't add support for children. It wasn't specifically asked for but I can.
To be fully featured it would need to recursively add annotation to all children that support it.
Alternatively I could just add it to the first child but not sure how useful that would be.

Reply:
@lhercot  - How do you feel about fully implementing annotation to all children that support it?

Reply:
We probably should but that might be complicated.

Reply:
@lhercot would you like me to implement it in this PR or add it to the backlog and do a separate PR?

Reply:
Turns out it wasn't as complex as I thought so I didn't do a separate PR. Now all children get annotation unless they already have annotation configured. I also added integration testing to verify annotation on child objects.